modules/ROOT/pages/supported-types-of-tests.adoc

```diff
@@ -0,0 +1,41 @@
+= MUnit Supported Types of Tests
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The following are definitions and concepts that MuleSoft recommends to take into account while designing and organizing test cases for your Mule applications. Understanding these definitions and concepts will help you build more robust MUnit tests by covering multiple edge cases, automate your tests through your CI/CD pipeline, and maximize the quality coverage for your Mule application.
+
+As a general good practice, MuleSoft recommends following a naming convention for file names and tests so that you can quickly realize what was being tested when a test fails.
+
+When testing a Mule application MUnit allows you to cover three different types of tests:
+
+* Unit tests.
+* Integration tests.
+* Functional test.
```

Comment:
OMIT in favor of introducing the three types right at the beginning, as suggested above.